docs/guide.mdx

```diff
@@ -0,0 +1,432 @@
+---
+sidebar_position: 10
+sidebar_label: "Walkthrough"
+id: deployment-guide
+title: Deployment Guide
+description: Deployment Guide
+slug: /walkthrough
+---
+
+This tutorial will show in detail how to:
+- [Configure a Kwil node](./walkthrough#configuration)
+- [Deploy a Kwil network with a standalone node](./walkthrough#deploy-a-standalone-kwil-node) 
+- [Deploy a Kwil network with multiple nodes](./walkthrough#deploy-a-multi-node-kwil-network)
+- [Add a node to the existing Kwil network](./walkthrough#join-kwil-network)
+- [Deploy databases](./walkthrough#deploy-database)
+- [Deploy local test networks](./walkthrough#test-networks)
+
+## Prerequisites
+
+In order to begin using Kwil, you will need the following:
+
+### Binaries
+
+Download the [latest release](https://github.com/kwilteam/binary-releases/releases/tag/v0.6.0-beta) of `kwild`, `kwil-admin` and `kwil-cli` binaries for your platform.
+
+### Docker
+
+To run Kwil nodes in a container:
+- Install [Docker engine](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/) specific to your platform.
+- Download `kwild` docker image:
+  ```bash
+  docker pull kwildb/kwil:latest
+  ```
+
+### Kuneiform Files
+Sample Kuneiform files can be viewed online at [https://ide.kwil.com](https://ide.kwil.com) under `sample_databases` folder. Refer to the [Kuneiform Guide](./category/kuneiform) to create your own Kuneiform schema files. 
+
+## Deploy a Standalone Kwil Node
+
+### Configuration
+
+Initialize the root directory with the required configuration files by running the below command.
+
+```bash
+$ kwil-admin setup init -o ~/.kwild
+Successfully initialized node directory: ~/.kwild
+```
+
+This generates the new node's private key, genesis and config files in the specified root directory.
+If the `-o` flag is not used, the default location is `./.testnet`.
+
+```bash
+$ tree ~/.kwild
+~/.kwild
+├── abci
+│   ├── config
+│   │   └── genesis.json
+│   └── data
+├── config.toml
+└── private_key
+
+4 directories, 3 files
+```
+
+The configuration is initially set to its default values. These can be updated manually or overridden using command line flags or environment variables. Refer to the [configuration guide](./daemon/kwild#configuration-override) for more information on this.
+
+Notable settings in `config.toml` and `genesis.json` are described below.
+
+#### Application Configuration (`config.toml`)
+
+This file contains operational parameters for the blockchain and databases. Some vital configurations include:
+
+- `app.grpc_listen_addr`: The address on which the user gRPC server listens (default is `localhost:50051`).
+- `app.http_listen_addr`: The address on which the HTTP gateway to the gRPC service listens (default `localhost:8080`).
+- `chain.p2p.listen_addr`: The node's P2P Default listening address is `0.0.0.0:26656`
+
+:::caution
+The current Kwil release does not require gas to execute transactions.
```

Comment:
I really think we can take this out.  Less is more; we should be giving minimal information in a walkthrough.